src/prelaunch_data_generator/cl_genesis/cl_genesis_data_generator.star

```diff
@@ -125,7 +123,7 @@ def generate_cl_genesis_data(
 
 	cl_genesis_generation_cmd = [
```

Comment:
perhaps this could go behind an if condition as well; if capella 0 then "capella" otherwise "merge"